<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Fsites</class>
 <widget class="QDialog" name="Fsites">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>792</width>
    <height>307</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Sites</string>
  </property>
  <property name="windowIcon">
   <iconset resource="rssources.qrc">
    <normaloff>:/images/logo_erp.png</normaloff>:/images/logo_erp.png</iconset>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>781</width>
     <height>291</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Sites de Production</string>
    </attribute>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>200</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Quitter</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ajouter_but_site">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>30</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Ajouter</string>
     </property>
    </widget>
    <widget class="QPushButton" name="refresh_but_site">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>90</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Rafraichir</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupe_emp_site">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>221</width>
       <height>81</height>
      </rect>
     </property>
     <property name="title">
      <string>Emplacement</string>
     </property>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Departement</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="dept_text_prod">
      <property name="geometry">
       <rect>
        <x>79</x>
        <y>50</y>
        <width>133</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="ville_text_prod">
      <property name="geometry">
       <rect>
        <x>79</x>
        <y>24</y>
        <width>133</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>24</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Ville</string>
      </property>
     </widget>
    </widget>
    <widget class="QComboBox" name="combo_nature_site">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>180</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Station Nucleaire</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Station Electrique</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Station Thermique</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>180</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Nature du site</string>
     </property>
    </widget>
    <widget class="QTableView" name="table_sites_prod">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>30</y>
       <width>291</width>
       <height>211</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Nom</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="nom_text_prod">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>133</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <zorder>groupe_emp_site</zorder>
    <zorder>pushButton_3</zorder>
    <zorder>ajouter_but_site</zorder>
    <zorder>refresh_but_site</zorder>
    <zorder>combo_nature_site</zorder>
    <zorder>label_6</zorder>
    <zorder>table_sites_prod</zorder>
    <zorder>label</zorder>
    <zorder>nom_text_prod</zorder>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Station de Stockage</string>
    </attribute>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>220</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Quitter</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupe_emp_stock">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>222</width>
       <height>79</height>
      </rect>
     </property>
     <property name="title">
      <string>Emplacement</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Ville</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="ville_text_stock"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Departement</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="dept_text_stock"/>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="ajouter_but_stock">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>30</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Ajouter</string>
     </property>
    </widget>
    <widget class="QPushButton" name="refresh_but_stock">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>90</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Rafrachir</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>200</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Capacit√©</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="qty_stock_text">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>200</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QComboBox" name="combo_nature_stock">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>170</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Import</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Export</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>170</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Nature de Stockage</string>
     </property>
    </widget>
    <widget class="QTableView" name="table_sites_stock">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>30</y>
       <width>301</width>
       <height>221</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="nom_text_stock">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>40</y>
       <width>133</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Nom</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="rssources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>tabWidget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>710</x>
     <y>262</y>
    </hint>
    <hint type="destinationlabel">
     <x>425</x>
     <y>228</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>Fsites</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>727</x>
     <y>263</y>
    </hint>
    <hint type="destinationlabel">
     <x>788</x>
     <y>190</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>Fsites</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>677</x>
     <y>239</y>
    </hint>
    <hint type="destinationlabel">
     <x>624</x>
     <y>297</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
